test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -150,6 +150,7 @@ func (s *packageBaseSuite) SetupSuite() {
 
 func (s *packageBaseSuite) RunInstallScript() {
 	// fixme: use the official install & remove manual creation of /etc/datadog-agent/datadog.yaml
+	s.Env().RemoteHost.MustExecute("sudo mkdir -p /etc/datadog-agent")
```

Comment:
@arbll Do we have a guarantee in the (customer) install script today that `/etc/datadog-agent` exists before installer install or should we add it in this PR?

Without it sockets paths will not be configured and instrumentation will fail.

Reply:
Why is that ? Don't we set the path in env variables ?

Reply:
Checked offline -- created the dir if it doesn't exist during installer setup in [c0b72c5](https://github.com/DataDog/datadog-agent/pull/25577/commits/c0b72c5b793ddc3403c6f15b4fa9d646f41fb291)